packages/frontend/web/components/ArticleBody.tsx

```diff
@@ -570,10 +570,9 @@ const ArticleBody: React.SFC<{
                         mainMedia,
                         pillarFigCaptionIconColor[CAPI.pillar],
                     )}
-                    dangerouslySetInnerHTML={{
-                        __html: CAPI.main,
-                    }}
-                />
+                >
+                    <ArticleRenderer elements={CAPI.mainMediaElements} />
```

Comment:
I didn't anticipate using the ArticleRenderer for media elements. Does this have any repercussions?

Reply:
Not sure. I guess `ArticleRenderer` is really too powerful for this task. Maybe it makes more sense to create a lighter `MainMediaRenderer` which acts on a more limited subset of Element types?

Reply:
Is it worth renaming ArticleRenderer anyway? It really is not a descriptive name (the entire service and most things in it are involved in article rendering!). `ElementRenderer` would be better and also help make this use case (for main images) more obviously okay.

Reply:
Ah wait! I just realised this is using code in `web`! That breaks one of our article principles - we don't share this kind of code really. I think this is why the naming confused me.

What about just copying this behaviour into our existing `Elements.tsx` file, or if the behaviour is special enough, just creating a `MainMedia` component in AMP?